docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -16,37 +19,100 @@ For scripts to successfully use BODL to access storefront data, the following mu
 ## BODL event schemas
 
 BODL exposes data when shoppers trigger storefront events. Currently, BODL supports the following storefront events:
-- Start checkout
-- Purchase order 
 
-### Start checkout event schema
+- [View product page](#view-product-page-event-schema)
+- [View product category](#view-product-category-event-schema)
+- [Add product to cart](#add-product-to-cart-event-schema)
+- [View cart](#view-cart-event-schema)
+- [Remove product from cart](#remove-product-from-cart-event-schema)
+- [Start checkout](#start-checkout-event-schema)
+- [Purchase order](#purchase-order-event-schema)
 
-When the shopper clicks on the button to initiate a checkout process, the following fields appear in the BODL object:
+### View product page
+
+When the shopper clicks a button to view a product page, the following fields appear in the BODL object for the item:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `channel_id` | integer | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
```

Comment:
@bc-rmalyavc Is my description correct (is `id` the `event_id`)? Sometimes I get an empty string (screenshot)
![view product page](https://user-images.githubusercontent.com/96258747/227020145-69655afa-052c-4228-bd6c-c1990df876e0.png)

Reply:
@bc-rmalyavc Yes, let us add them in. I don't know why they were left out.

Reply:
Engineering ticket that I'm following to get this fixed: [DATA-11047](https://bigcommercecloud.atlassian.net/browse/DATA-11047)